packages/contracts/package.json

```diff
@@ -46,6 +47,7 @@
     "ethers": "^4.0.17",
     "ganache-cli": "6.1.8",
     "openzeppelin-eth": "^2.0.2",
+    "remixd": "^0.1.8-alpha.6",
```

Comment:
seems like more of a custom dev environment thing than a necessary dev-dependency for the package?

Reply:
@snario I believe it's still a valid use case for a `devDependency`; take TypeScript, for example. You won't import it anywhere in your app, you won't make any reference to it beyond a run-script, but you need to install it as a package dependency.

Reply:
I agree for TypeScript but in this case `remixd` is something you would use only if you use Remix IDE for writing Solidity and like to map your file system to that website. It feels more like `.vscode` or something like this: https://www.npmjs.com/package/jetbrains

Reply:
@snario I can remove it. Just wanted a quick way to run remixd as a script inside the project. I could probably achieve the same with a powershell, or similar, alias.